src/Button/Button.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+require('./Button.less');
+
+class Button extends React.Component {
```

Comment:
I know this is simple but can we have doc comments to explain what these actually are, what props they can take, what state they have, etc?

Additionally, maybe this would be a good time to actually use [React `propTypes`](https://facebook.github.io/react/docs/reusable-components.html).